app/views/rescue_codes/new.html.slim

```diff
@@ -0,0 +1,21 @@
+- title t('titles.rescue_codes')
+
+h1.heading = t('headings.rescue_codes')
+p
+  | Below are a set of rescue codes that we have generated for you.
+    You can use these codes to access your account in case you loose
+    access to your phone. Please download these codes and keep them
+    in a safe place.
```

Comment:
I can see how that's practical, but I don't think we want to encourage people to leave the codes on their computer. I vote for being secure by default and not giving the option to download them at all. We need strong messaging that states the importance of writing the codes down or printing them right away.